snapcraft/internal/lxd.py

```diff
@@ -29,7 +29,7 @@
 import yaml
 
 from snapcraft.internal.errors import SnapcraftEnvironmentError
-from snapcraft.internal import common
+from snapcraft.internal.common import format_snap_name, is_snap
```

Comment:
@kalikiana this is how you can patch the common.is_snap imported in lxd:
http://paste.ubuntu.com/24961616/